lib/location/Kconfig

```diff
@@ -77,6 +81,17 @@ config LOCATION_METHOD_GNSS_VISIBILITY_DETECTION_SAT_LIMIT
 
 endif # LOCATION_METHOD_GNSS
 
+if LOCATION_METHOD_CELLULAR
+
+config LOCATION_METHOD_CELLULAR_EXTERNAL
+	bool "Requesting resolution to neighbor cell information handled by the application outside of Location library."
```

Comment:
That is not a true or false sentence. Please clarify.

Reply:
Not sure how these should be written as trying to find examples from nRF looks equally bad :)
This is what I'm adding now:
`Resolution to neighbor cell information is handled by the application outside of Location library`
Please suggest a change.

Reply:
This was clarified in offline discussion.
Enabled: Location library requests application to resolve neighbor cell information into a location
Disabled: Location library will send a request to selected cloud service to resolve location based on neighbor cell information

It's similar config as LOCATION_METHOD_GNSS_AGPS_EXTERNAL and LOCATION_METHOD_GNSS_PGPS_EXTERNAL are for GNSS assistance.
LOCATION_METHOD_CELLULAR determines if cellular location is done at all, that is, NCELLMEAS sent to modem.

I'll close this comment.